OTAnalytics/plugin_parser/otvision_parser.py

```diff
@@ -341,6 +391,65 @@ def __str__(self) -> str:
 )
 
 
+# TODO Review: these methods for creating PythonDetections and PythonTracks are static
+# TODO and could live outside a class for reusability in streaming_parser
+def parse_python_detection(
```

Comment:
@briemla  Review: these methods for creating PythonDetections and PythonTracks are static and could live outside a class for reusability in streaming_parser